src/components/sendWritable/send.js

```diff
@@ -82,6 +83,16 @@ class SendWritable extends React.Component {
             <h2>{this.props.t('Transfer')}</h2>
             <span className={`${styles.subTitle} ${styles.transfer}`}>{this.props.t('Quickly send and request LSK token')}</span>
           </header>
+
+          <div className={`${grid.row} ${styles.tab} `}>
+            <div className={`${grid['col-xs-6']} ${styles.tabActive} send-tab`}>
+              {this.props.t('Send')}
+            </div>
+            <div className={`${grid['col-xs-6']} ${styles.tabInactive} request-tab`} onClick={() => { this.props.setTabSend(false); }}>
+              {this.props.t('Request')}
+            </div>
+          </div>
```

Comment:
The tab switcher is duplicated in send and receive. Should be a separate component with own CSS.